### ISSUE TITLE
shuvlord -j nukes ihpc servers

### ISSUE THREAD

**Issue by maxlcummins** [maxlcummins/pipelord_old#9]
Multithreading shuvlord requests 32gb and 16(?) threads per shovill job, so -j requests a crazy amount of resources not available on most clusters and causes admin issues and job failure

**Comment by Tu6ka** [comment]
Have you tried `snakemake --jobs 2`??? so only 2 instances would work, or for clusters with more memory, I guess biggest is 128gb and you can run 4 jobs at the same time.

I did not used it by myself, but I guess this should help.

**Comment by maxlcummins** [comment]
yeah we can limit the paralellisation of the snakemake or we could do something a little bit more nuanced with the parameters of the actual shovill job being run. Ideally we can -j everything and use the maximum number of available threads with our snakemake runs.

**Comment by Tu6ka** [comment]
I guess we can add `threads = snakemake.utils.available_cpu_count()/8` or something similar into the rule.

Also need to add something to deal with RAM usage. 
BTW by default (which seems to be the case) shovill use 8 threads, 16gb of ram.

P.S. For iHPC it should work, as far as I remember all nodes have >= 16gb per 8 cores, just maybe add `int` in front of cpu count as I'm not sure how it will behave with floating point numbers.

**Comment by Tu6ka** [comment]
should be solved for development branch

**Comment by maxlcummins** [comment]
Thanks Dmi!
